walkthroughs/howto-ingress-gateway/infrastructure/ecs-service.yaml

```diff
@@ -590,7 +589,6 @@ Resources:
         - Name: 'cw-agent'
           Image: !Sub '${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/${CloudWatchAgentImageName}'
```

Comment:
In that case, we could just use `amazon/cloudwatch-agent` from docker hub.
Basically reverting PR https://github.com/aws/aws-app-mesh-examples/pull/282

Reply:
But if we don't run this as user `1337` won't the Envoy double count the metrics?

Reply:
Discussed over Chime. In the colorgateway ECS task, Envoy is running as a standalone proxy. The traffic leaving the cw-agent container will not be redirected, so the metrics will be unaffected.